aten/src/ATen/mps/MPSProfiler.h

```diff
@@ -51,15 +50,7 @@ struct BaseInfo {
   // handle used to identify the profile info's instance (usually the pointer)
   const uintptr_t handle;
 
-  virtual const std::string toString(double gpuTime = 0, double schedulingTime = 0) const {
-    // the gpuTime will be non-zero mainly for event-based signposts.
-    // The interval-based signposts will have "duration" as well as accumulated
-    // total GPU time, up to the point of execution.
-    return fmt::format("{}{}",
-                       gpuTime > 0.0 ? fmt::format(", gpu={:.3f} ms", gpuTime) : "",
-                       schedulingTime > 0.0 ? fmt::format(", cpu={:.3f} ms", schedulingTime) : "");
-  }
-
```

Comment:
Moved functions that use the fmt lib to `MPSProfiler.mm` since the fmt lib is not packed into the torch wheel, and it thus will not be found in the torchvision c++ extension.